xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp

```diff
@@ -110,14 +112,13 @@ void CGUIDialogPVRTimerSettings::SetTimer(CFileItem *item)
   m_timerType     = m_timerInfoTag->GetTimerType();
   m_bIsRadio      = m_timerInfoTag->m_bIsRadio;
   m_bIsNewTimer   = m_timerInfoTag->m_state == PVR_TIMER_STATE_NEW;
-  m_bStartAnytime = m_bIsNewTimer || m_timerInfoTag->IsStartAtAnyTime();
-  m_bEndAnytime   = m_bIsNewTimer || m_timerInfoTag->IsEndAtAnyTime();
   m_bTimerActive  = m_bIsNewTimer || m_timerInfoTag->IsActive();
+  m_bStartAnyTime = m_timerInfoTag->m_bStartAnyTime;
+  m_bEndAnyTime   = m_timerInfoTag->m_bEndAnyTime;
```

Comment:
I'd like to keep the <code>m_bIsNewTimer || </code> as it was before.

If a new timer gets created, Kodi must define some meaningful default values. Let's assume a user manually (not from epg grid) creates a new repeating epg based timer that supports "any time". I think in most cases preselected "any time" is a better approach than concrete times for a first shot of a working the "query". Less klicks - more fun.